site/src/pages/docs/server-multipart.mdx

```diff
@@ -0,0 +1,145 @@
+# Handling a multipart request
+
+Armeria provides <type://Multipart> encoder and decoder built on top of 
+[Reactive Streams](https://www.reactive-streams.org/).
+
+## Building a multipart request
+
+A <type://Multipart> consists of multiple <typeplural://BodyPart>.
+Each <type://BodyPart> is divided into a headers and a body. A <type://BodyPart> headers can be created simply 
+using <type://ContentDisposition>. 
+`String`, <type://HttpData>, or <type://StreamMessage> can be set as the body of the <type://BodyPart>.
+
+<Tip>
+
+You can also use <type://HttpHeaders> to set a complex headers into <type://BobyPart#headers()>.
```

Comment:
since headers is plural, using "set" with "into" sounds unnatural

```suggestion
You can also use <type://HttpHeaders> to set complex headers for <type://BobyPart#headers()>.
```